scripts/ci/run-jenkins.sh

```diff
@@ -227,6 +227,7 @@ if [[ ${CI_TAGS} == *'webassembly'* ]] || [[ ${CI_TAGS} == *'wasm'* ]];
             ${TESTCMD} --label=v8-system-core --timeout=60m make -C sdks/wasm run-v8-system-core
             ${TESTCMD} --label=sm-system-core --timeout=60m make -C sdks/wasm run-sm-system-core
             ${TESTCMD} --label=jsc-system-core --timeout=60m make -C sdks/wasm run-jsc-system-core
+            ${TESTCMD} --label=v8-corlib --timeout=60m make -C sdks/wasm run-v8-corlib
```

Comment:
Will the tests only work for v8 or is this just to test things out?

Reply:
Just a test.